### ISSUE TITLE
used parameter reported as unused in template loop

### ISSUE THREAD

**Issue by rijulg** [vuejs/eslint-plugin-vue#543]
**Tell us about your environment**

* **ESLint Version:** ^3.0.0-rc.10
* **eslint-plugin-vue Version:** vue/strongly-recommended
* **Node Version:** v10.3.0

**Please show your full configuration:**

```
module.exports = {
  root: true,
  env: {
    node: true,
    jest: true,
  },
  'extends': [
    'plugin:vue/strongly-recommended',
    '@vue/airbnb'
  ],
  rules: {
    "vue/no-parsing-error": [2, {"x-invalid-end-tag": false}],
    "no-param-reassign": [2, { "props": false }],
    'no-console': process.env.NODE_ENV === 'production' ? 'error' : 'off',
    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
    'import/no-extraneous-dependencies': process.env.NODE_ENV === 'production' ? 'on' : 'off',
  },
  parserOptions: {
    parser: 'babel-eslint'
  },
}
```

**What did you do? Please include the actual source code causing the issue.**

```vue
<MenuItem
  v-for="{name, icon} in menu"
  :name="name"
  :key="name">
  <Icon
      v-if="icon"
      :type="icon"/>
  <span>{{ name }}</span>
</MenuItem>
```

**What did you expect to happen?**
no errors

**What actually happened? Please include the actual, raw output from ESLint.**

> Module Warning (from ./node_modules/eslint-loader/index.js):
error: 'icon' is defined but never used (vue/no-unused-vars) at src\components\sidebar\sidebar.vue:16:23:
  14 |
  15 |       <MenuItem
> 16 |         v-for="{name, icon} in menu"
     |                       ^
  17 |         :name="name"
  18 |         :key="name">
  19 |
error: Expected indentation of 8 spaces but found 10 spaces (vue/html-indent) at src\components\sidebar\sidebar.vue:21:1:
  19 |
  20 |         <Icon
> 21 |           v-if="icon"
     | ^
  22 |           :type="icon"/>
  23 |         <span>{{ name }}</span>
  24 |

Besides the loop error, can someone please explain why indenting the divs inside the main div is reported as incorrect? I am just not sure if indenting them is right or not anymore but I thought it should have been.

**Comment by mysticatea** [comment]
Thank you for the report.

This looks like the same issue as #423. It was fixed on the next version.
Could you try the current beta version of `eslint-plugin-vue`?

**Comment by michalsnik** [comment]
I tested it on `eslint-plugin-vue@next` and it seems to be fixed.
